cmd/provisioner-localpv/app/provisioner_hostpath.go

```diff
@@ -96,16 +97,21 @@ func (p *Provisioner) ProvisionHostPath(opts pvController.VolumeOptions, volumeC
 	//labels[string(v1alpha1.StorageClassKey)] = *className
 
 	//TODO Change the following to a builder pattern
-	pvObj, err := persistentvolume.NewBuilder().
+	pvBuilder := persistentvolume.NewBuilder().
 		WithName(name).
 		WithLabels(labels).
 		WithReclaimPolicy(opts.PersistentVolumeReclaimPolicy).
 		WithAccessModes(pvc.Spec.AccessModes).
 		WithVolumeMode(fs).
 		WithCapacityQty(pvc.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)]).
 		WithLocalHostDirectory(path).
-		WithNodeAffinity(nodeHostname).
-		Build()
+		WithNodeAffinity(nodeHostname)
+
+	if shared == true {
```

Comment:
Related to the above comment on boolean variable naming, the above can be changed to `if isShared {`